recipe/meta.yaml

```diff
@@ -50,12 +50,13 @@ about:
 
 extra:
   recipe-maintainers:
+    - c-bata
+    - contramundum53
     - g-votte
     - HideakiImamura
+    - himkt
     - hvy
     - iwiwi
+    - knshnb
```

Comment:
I updated the maintainer list. The corresponding change can be seen in README.md.

Reply:
To update the README, please [re-render]( https://conda-forge.org/docs/maintainer/infrastructure.html#conda-forge-admin-please-rerender )

Reply:
To remove irrelevant commits, I'm working on the release in a new PR https://github.com/conda-forge/optuna-feedstock/pull/32.